tasks/affective_task/utils/display_msg_affective_disscussion.py

```diff
@@ -0,0 +1,12 @@
+import pygame
+from config import UPDATE_RATE
+from common import render_text_center
+
+def display_msg_affective_disscussion(screen ,msg: str,milliseconds: int):
```

Comment:
space after the comma, not before.

 `def display_msg_affective_disscussion(screen, msg: str, milliseconds: int):`